install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -312,8 +314,6 @@ data:
 {{- end }}
 
 {{- if .Values.gke.enabled }}
-
-  ipam: "kubernetes"
```

Comment:
Does removing this line introduce a risk of selecting the wrong IPAM for GKE if user does not specify any from the command line (or if they use any other than `kubernetes`)? If so, one alternative could be to change the check below to make it depend on `.Values.gke.enabled`?

Reply:
This approach here follows what we discussed in the Cilium Community meeting on Monday, i.e. that we expect users to set IPAM explicitly.